### PR TITLE
Upgrade Ruby version

### PR DESCRIPTION
**Pull request by jackscotti**
So that we use a version available from the VM

### CODE REVIEW

**Comment by rboulton** [comment]
2.2.3 is not a supported heroku ruby version: https://devcenter.heroku.com/articles/ruby-support#ruby-versions.  Is 2.2.4-p173 available in the VM?

**Comment by jackscotti** [comment]
Good point. 

These are the versions available on my VM:
  1.9.3
  1.9.3-p484
  1.9.3-p550
  2.1
  2.1.2
  2.1.4
  2.1.5
  2.1.6
  2.1.7
  2.2
  2.2.2
  2.2.3

These the versions supported by Heroku:
1.8.7 : patchlevel 376, Rubygems : 1.8.24
1.9.2 : patchlevel 327, Rubygems : 1.3.7.1
1.9.3 : patchlevel 551, Rubygems : 1.8.23.2
2.0.0 : patchlevel 648, Rubygems : 2.0.14.1
2.1.8 : patchlevel 400, Rubygems : 2.2.5
2.2.4 : patchlevel 173, Rubygems : 2.4.5.1
2.3.0 : patchlevel 0, Rubygems : 2.5.1

There isn't a single shared one. I will talk to the infrastructure team.

**Comment by jackscotti** [comment]
2.3.0 is available now and 2.4.0 will be available soon. Updating this PR to use 2.3.0.

**Comment by deanwilson** [comment]
Just to clarify - 2.3.0 is done and I'm currently working on 2.2.4, not 2.4.0.

**Comment by jackscotti** [comment]
@rboulton updated.

@deanwilson thanks for clarifying, I mixed the numbers!

File: Gemfile [hunk 1]
```diff
@@ -26,6 +25,7 @@ group :test, :development do
   gem 'timecop', '~> 0.5'
   gem 'capybara'
   gem 'byebug'
+  gem 'pry'
```
Comment (malcolmbaig) [inline Gemfile]: a trifling thing, but if you just amend the entry above this one to `pry-byebug` you'll get pry down as a dependency anyway.

**Comment by malcolmbaig** [comment]
some proper minor stuff - otherwise LG. shout when updated and i'll merge.

**Comment by jackscotti** [comment]
@mobaig updated.

**Pull request merged**